specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/GetProjectDetailsMySqlSqlTask.json

```diff
@@ -0,0 +1,46 @@
+﻿{
```

Comment:
I don't think this file is required. It is used for the private preview MySQL scenario only.

Reply:
Yeah, the objects here don't look like they are referenced anywhere.

Reply:
My latest commit should have removed this file from the PR.